dotCMS/src/main/java/com/dotcms/vanityurl/model/CacheVanityKey.java

```diff
@@ -0,0 +1,24 @@
+package com.dotcms.vanityurl.model;
+
+import com.dotcms.repackage.org.apache.commons.collections.keyvalue.MultiKey;
+
+import java.io.Serializable;
+
+/**
+ * Encapsulates the keys for cache vanities.
+ * @author jsanca
+ */
+public class CacheVanityKey extends MultiKey implements Serializable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "Serializable" is implemented by a super class; there is no need to implement it here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1939)